kpet/tree.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2018 Red Hat, Inc. All rights reserved. This copyrighted
+# material is made available to anyone wishing to use, modify, copy, or
+# redistribute it subject to the terms and conditions of the GNU General Public
+# License v.2 or later.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+"""Module where the `tree` command is implemented"""
+from __future__ import print_function
+import os
+from kpet.exceptions import ActionNotFound
+
+
+class TemplateDirNotFound(Exception):
+    """Raised when template dir is not found under the dbdir"""
+
+
+def list_tree(args):
+    """List all the templates a.k.a trees available on the current db"""
```

Comment:
Could you please break out the `args` into real parameters, and document them? I.e. have an actual `db` parameter? It's better to be explicit and clear about what inputs a function needs (and make it require only what's needed). Otherwise it degenerates into a global state being passed around everywhere, like in skt.